wled00/data/settings_wifi.htm

```diff
@@ -6,18 +6,100 @@
 	<meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" name="viewport"/>
 	<title>WiFi Settings</title>
 	<script>
-		var d=document;
-		var loc = false, locip;
+		let d = document;
+		let loc = false, locip;
+		let scanLoops = 0;
+
+		function Local() {
```

Comment:
No added benefit of using function here except wasting a few bytes of flash memory. Better to leave the code where it was in `S()`.

Reply:
Ooops, this was left over from my first attempt where i had to call it in two places, moving back ;).